lib/main_common.pm

```diff
@@ -176,6 +205,19 @@ sub is_bridged_networking {
     return $ret;
 }
 
+sub is_livesystem {
+    return (check_var("FLAVOR", 'Rescue-CD') || get_var("LIVETEST"));
+}
+
+sub is_kde_live {
+    return get_var('FLAVOR', '') =~ /KDE-Live/;
+}
+
+# TODO check why we want this enabled on SLE staging but not openSUSE staging
+sub packagekit_available {
+    return !check_var('FLAVOR', 'Rescue-CD') && (!is_staging || is_sle);
```

Comment:
@DimStar77 @lnussel this differs between SLE and openSUSE and causes the "glxgears" test to not run on openSUSE staging tests but on SLE. "glxgears" is important for SLE because it is the only packagekit based install test in staging tests. Any reason why we should not enable this on openSUSE as well?

Reply:
Generally speaking, we should be able to cope with glxgears in Stagings (Staging2 only, or Mesa-demo would need to move to Ring1) - at least for TW I'll have to add Mesa-demo to the test DVD, but that cost is acceptable.

Reply:
Done in https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/4443 , based on this PR here. http://lord.arch/tests/606#step/glxgears/12 shows Mesa-demo-x missing. Can you add it?